graphql/fragments/ContentNode.graphql

```diff
@@ -3,6 +3,7 @@
 fragment ContentNodeFields on ContentNode {
 	id
 	... on NodeWithContentEditor {
+		schemaOrgMeta
```

Comment:
This is a problematic addition given we don't always expose this field...

Reply:
Hmmm, we might want to introduce our own interface that always gets registered (`NodeWithSchemaOrgMeta`), so that this query is always valid. If Parse.ly is installed, then we can direct the node to implement that interface and return data. If not, it returns null.